backend/benefit/applications/tests/conftest.py

```diff
@@ -114,6 +115,7 @@ def association_application(mock_get_organisation_roles_and_create_company):
     application.company = mock_get_organisation_roles_and_create_company
     application.save()
     application.company.company_form = "association"  # TODO: fix with actual value
```

Comment:
Is the TODO on this line still valid?
If it is do you now know the actual value that would fix this?